index.html

```diff
@@ -285,6 +285,33 @@ <h1>Skills</h1>
         </div>
     </section> 
     
+    <section class="contact-form">
+        <h1 class="title-contact">Contact Me</h1>
+        <h2 class="view">If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I’d love to help with it 
+        </h2>   
+        
+        <form>
+            <input type="text" class="input" placeholder="Full name" maxlength="30">
+            <br>
+            <br>
+            <input type="email" class="input" placeholder="Email address">
+            <br>
+            <br>         
+            <input type="text" class="box" placeholder="Enter text here" maxlength="500">
+            <br>
+            <br>
+            <div class="get-in-touch">
+                <button type="button"> Get In Touch </button>
+            </div> 
```

Comment:
- [ ] You should implement the button interactions (enable, hover, etc.).
- [ ] The submit button can be either an `input` element with type `submit` or a `button` element with type `submit` or with no `type` attribute. Please fix it.

![image](https://user-images.githubusercontent.com/38635158/149846211-efd031d5-3e87-42c7-8801-1b249096d6ba.png)

Reply:
button attribute added